### PR TITLE
Smarty variables]  Remove isset from batch form

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Affects url civicrm/financial/financialbatches?reset=1&batchStatus=1

Before
----------------------------------------
fatal with hardened smarty

After
----------------------------------------
ok

Technical Details
----------------------------------------

Comments
----------------------------------------

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
Stacktrace
api\v4\Action\DateTest::testRelativeDateRanges
Failed asserting that an array contains 9.

/home/jenkins/bknix-dfl/build/core-22187-36q9f/web/sites/all/modules/civicrm/tests/phpunit/api/v4/Action/DateTest.php:96
/home/jenkins/bknix-dfl/extern/phpunit8/phpunit8.phar:671

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
@colemanw this is failing only on a known test fail - should be an easy one

**Comment by demeritcowboy** [comment]
jenkins retest this please

File: templates/CRM/Financial/Form/Search.tpl [hunk 1]
```diff
@@ -9,7 +9,7 @@
 *}
 
 {* Financial search component. *}
-{if !isset($batchStatus)}
+{if $batchStatus}
```
Comment (demeritcowboy) [inline templates/CRM/Financial/Form/Search.tpl:12]: Should be `!$batchStatus`

**Comment by eileenmcnaughton** [comment]
@demeritcowboy I've updated this - on review I've changed the approach slightly

**Pull request merged**